CHANGELOG-prerelease.md

```diff
@@ -5,7 +5,8 @@
 - [0354352](https://github.com/patternfly/patternfly-elements/commit/03543523064f4a885b44bd334b0e24a77f00aee9) feat: Open a pull request from the command-line
 - [7559485](https://github.com/patternfly/patternfly-elements/commit/755948553015d8745f8faad8b7a299031ec6fff3) feat: Migrate from Travis to GitHub Actions
 - [51d055a](https://github.com/patternfly/patternfly-elements/commit/51d055ae15b3570c6d902c60d45163a0154e1011) feat: Add automerge action for pull requests when their base PR is updated
-- [](https://github.com/patternfly/patternfly-elements/commit/) fix: Update baseline snapshots to cover full page
+- [9501eb](https://github.com/patternfly/patternfly-elements/commit/9501ebb87cfc371563c04ba155a101846fe26e20) fix: Update baseline snapshots to cover full page
+- [](https://github.com/patternfly/patternfly-elements/commit/) fix: pfe-tabs + vertical display on mobile
```

Comment:
When you merge the latest from master, there is now a 1.x Changelog! Can you move this entry there?